### PR TITLE
feat(cdk): schematics shouldn't be a direct depended on cdk

### LINKED ISSUE #2413: 🤖 - `nx test-schematics cdk` should not rebuild cdk
### Description

 `nx test-schematics cdk` should not rebuild cdk

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #2413

## What is the new behavior?

```
$ rm -rf dist
$ nx test-schematics cdk
```

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by nsbarsukov — APPROVED** [review]

File: scripts/postbump.ts [hunk 1]
```diff
@@ -7,7 +7,16 @@ import {syncVersions} from './shared/sync-versions';
 
 (function main(): void {
     syncVersions([`./projects`, `./package-lock.json`], version);
-    overwriteVersion(resolve(`./projects/core/constants/version.ts`), version);
+
+    // TODO: remove in v3.0
+    overwriteVersion(resolve(`./projects/core/constants/version.ts`), `VERSION`, version);
```
[the remark this answers is not in the document]
Comment (splincode) [reply scripts/postbump.ts]: changed

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/taiga-schematics/schematics/ng-add/constants/version.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/**
+ * @note:
+ * projects/cdk/constants/version.ts is not under 'rootDir'
+ * workaround for success compiling without cdk dependencies
+ */
+export const TUI_SCHEMATICS_VERSION = `2.60.0`;
```
[the remark this answers is not in the document]
Comment (nsbarsukov) [reply projects/taiga-schematics/schematics/ng-add/constants/version.ts:6]: May be we should remove nesting inside this schematics-package ?

I am not sure that we need folder `schematics` inside package `taiga-schematics`

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/core/constants/version.ts [hunk 3]
```diff
@@ -1 +1,7 @@
-export const VERSION = `2.60.0`;
+/**
+ * @deprecated: use {@link TUI_VERSION}
+ * TODO: remove in v3.0
+ */
+import {TUI_VERSION} from '@taiga-ui/cdk/constants/version';
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/core/constants/version.ts]: fixed

File: projects/taiga-schematics/schematics/ng-add/constants/version.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (splincode) [reply projects/taiga-schematics/schematics/ng-add/constants/version.ts]: fixed

**Review by nsbarsukov — CHANGES_REQUESTED** [review]
Probably, you should also change this line

https://github.com/Tinkoff/taiga-ui/blob/6c13da56c6ae928db314bfe21321779d7b9dfb39/projects/taiga-schematics/package.json#L24

Screenshot 2022-08-23 at 10 35 58

**Comment by nsbarsukov** [comment]
@vladimirpotekhin review again, please 

(I hope we didn't break something :peka:)

**Comment by splincode** [comment]
@nsbarsukov @vladimirpotekhin fixed

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**